BasicTests/txtest.json

```diff
@@ -1,24 +0,0 @@
-[
-    {
```

Comment:
I have no idea how many teams are still using this files. perhaps it is better to leave it here as it is small